src/PaymentGateways/PayPalCommerce/PayPalCommerce.php

```diff
@@ -160,6 +160,16 @@ public function getOptions()
                 ],
                 'default' => 'donation',
             ],
+            [
+                'name' => esc_html__('Accept Venmo', 'give'),
+                'id' => 'paypal_commerce_accept_venmo',
+                'type' => 'radio_inline',
+                'default' => 'disabled',
+                'options' => [
+                    'enabled' => esc_html__('Enabled', 'give'),
+                    'disabled' => esc_html__('Disabled', 'give'),
+                ],
+            ],
```

Comment:
Clarifying question: can admins with a PayPal account from outside of the US accept payments through Venmo?